controllers/secretscope_controller_databricks.go

```diff
@@ -152,25 +152,66 @@ func (r *SecretScopeReconciler) submitACLs(instance *databricksv1alpha1.SecretSc
 	return nil
 }
 
-func (r *SecretScopeReconciler) submit(instance *databricksv1alpha1.SecretScope) error {
+// checkCluster checks if Databricks cluster supports ACLs, and checks if secret scope exists.
+func (r *SecretScopeReconciler) checkCluster(instance *databricksv1alpha1.SecretScope) error {
 	scope := instance.ObjectMeta.Name
 	initialManagePrincipal := instance.Spec.InitialManagePrincipal
```

Comment:
InitialManagePrincipal is optional, can we have a null check?
`InitialManagePrincipal string              `json:"initial_manage_permission,omitempty"``

Reply:
I've created a new issue for this: #125.